src/command.js

```diff
@@ -27,11 +29,29 @@ const Command = class {
 
   static create (rawInputs) {
     const command = doNotAllowMissingProperties(new this(rawInputs))
+    this._sanitizeCreation(command)
     this._validateCreation(command)
+
     return command
   }
 
+  // If there are extra validations, put them here.
   static _validateCreation (newlyCreatedCommandInstance) {
+    this.validateThatSchemaMustBeStatic(newlyCreatedCommandInstance)
+  }
+
+  // If there are extra sanitization processes, put them here.
+  static _sanitizeCreation (newlyCreatedCommandInstance) {
+    this._sanitizeEmptySchema(newlyCreatedCommandInstance)
+  }
+
+  static validateThatSchemaMustBeStatic (newlyCreatedCommandInstance) {
+    if (newlyCreatedCommandInstance._staticSchemaAttributeWasOverwritten) {
+      throw new CommandWithNonStaticSchemaError('Schema must be a static property -- not an instance property.')
```

Comment:
I think I'd rather use the word "assert" here to avoid confusion with "validate" in the Command class which means "add to the errors collection" and not "throw."

Also, if we're going to use the `_` prefix convention for private methods/fields, then we should make this method private. I'd recommend `_assertStaticSchema` or `_assertSchemaIsStatic`.